rust/lance-core/src/utils/tokio.rs

```diff
@@ -35,6 +35,7 @@ lazy_static::lazy_static! {
         .worker_threads(1)
         // keep the thread alive "forever"
         .thread_keep_alive(Duration::from_secs(u64::MAX))
+        .enable_all()
```

Comment:
Why would the CPU runtime need the I/O subsystem enabled?

Reply:
They have a mix of IO and CPU. I'm not sure how to separate them at this level. Should I just use the current runtime instead of the CPU runtime?